### ISSUE TITLE
Add onboard and swap analytics events

### ISSUE THREAD

**Issue by johnsmith-gooddollar** [GoodDollar/GoodProtocolUI#295]
- [ ]  swap_start / swap_success on celo (todo @L03TJ3 verify with ubeswap dev if this can be added)
- [x]  wallet_connect_start/success (discuss with @drptiago )

1. swap components is all legacy in sdk-v1. Ubeswap is not yet moved to good-design since there are not many style options anyway. @L03TJ3 is sending a message now ube devs to see if we can do something around events

2. wc events implementation is blocked by https://github.com/GoodDollar/GoodProtocolUI/issues/277

**Comment by johnsmith-gooddollar** [comment]
- `wallet_connect_start` was already added by @L03TJ3 
- `wallet_connect_success` doesn't requires changes at monorepo / additional callbacks to be added

https://github.com/GoodDollar/GoodProtocolUI/commit/4108ee794b48e7520c6a6a1cde88e41f814644e5

**Comment by johnsmith-gooddollar** [comment]
@L03TJ3 awaiting updates from ubeswap

**Comment by johnsmith-gooddollar** [comment]
@L03TJ3 we need to ask them for

a) flag to disable sending google analytics by their components/hooks etc
b) add callback props/params inside hooks/components for getting some events (including  swap_start / swap_success)

**Comment by johnsmith-gooddollar** [comment]
Awaiting updates from ubeswap. putting on hold by now
